src/code/PublishPSResource.cs

```diff
@@ -628,7 +631,7 @@ private bool IsValidModuleManifest(string moduleManifestPath)
                 <owners>Microsoft,PowerShell</owners>
                 <requireLicenseAcceptance>false</requireLicenseAcceptance>
                 <license type="expression">MIT</license>
-                <licenseUrl>https://licenses.nuget.org/MIT</licenseUrl>
+                <license>https://licenses.nuget.org/MIT</license>
                 <icon>Powershell_black_64.png</icon>
                 <projectUrl>https://github.com/PowerShell/PowerShell</projectUrl>

```

Comment:
Per the comment about naming guideline for the 3 (license, icon, project) would `projectUrl` change to the below? Or would icon be `iconUrl` and only license is named `license`?
```suggestion
                <project>https://github.com/PowerShell/PowerShell</project>

```

Reply:
Nope, `projectUrl` would not change to `project`, see: https://docs.microsoft.com/en-us/nuget/reference/nuspec#projecturl